GhidraPython/goresym_rename.py

```diff
@@ -0,0 +1,174 @@
+#This script labels Go stripped binaries in Ghidra based on GoReSym output
+#@category Analysis
+
+from ghidra.program.model.symbol.SourceType import USER_DEFINED
+
+import json
+import re
+
+# Maps entry function to the longest _rt0_ function found by GoReSym
+CHOICE_EST_ENTRY_ADDR = "entry to _rt0_ function mapping"
+# Maps pclntab section to TabMeta VA mapping, useful only when pclntab section is present
+CHOICE_EST_PCLNTAB = "pclntab to TabMeta VA mapping"
+# Tries to map addresses against known function names in Ghidra with functions found by GoReSym
+CHOICE_EST_KNOWN_FUNCS = "known function names mapping"
+# All available choices
+CHOICES = [CHOICE_EST_ENTRY_ADDR, CHOICE_EST_PCLNTAB, CHOICE_EST_KNOWN_FUNCS]
+
+fm = currentProgram.getFunctionManager()
+
+def iterable(obj):
+    if obj is None:
+        return False
+
+    try:
+        iter(obj)
+    except:
+        return False
+
+    return True
+
+def _entry_addr_estimator(hints):
+    if not iterable(hints["StdFunctions"]):
+        print("No StdFunctions present, required for entry addr estimation")
+        return None
+
+    # StdFunctions should have _rt0_[ARCH]_[OS](_lib)?, this maps to entry
+    candidates = [f for f in hints["StdFunctions"] if "_rt0_" in f["FullName"]]
+
+    if len(candidates) == 0:
+        print("No rt0 functions found")
+        return None
+
+    func = candidates[0]
+    if len(candidates) > 1:
+        # Just take the longest _rt0_ candidate
+        for c in candidates[1:]:
+            if len(c["FullName"]) > len(func["FullName"]):
+                func = c
+
+    for f in fm.getFunctions(True):
+        if f.name == "entry":
+            return f.entryPoint.offset-func["Start"]
```

Comment:
```suggestion
            return f.entryPoint.offset - func["Start"]
```

sorry, formatting nit. i almost thought the `-` was a `.` and it confused me.

Reply:
optional to fix this, i see this is your formatting preference throughout the file